src/app/zap-templates/templates/app/MatterIDL.zapt

```diff
@@ -5,10 +5,8 @@
 {{>idl_structure_definition extraIndent=0}}
 
 {{/chip_shared_structs}}
-{{!TODO: consider #chip_clusters as iteration point as well.
-{{!      Unsure about the differences}}
-{{#all_user_clusters}}
-  {{~side}} cluster {{asUpperCamelCase name}} = {{code}} {
+{{#chip_clusters}}
```

Comment:
What is the reason for switching to chip_clusters here?

Reply:
The comment said to consider switching and generated output seemed identical.

Reply:
Updated based on comment from @bzbarsky-apple as well ... I did not know we wanted to get rid of chip_clusters.